packages/clerk-js/src/ui/components/SignIn/SignInFactorTwoBackupCodeCard.tsx

```diff
@@ -25,14 +25,14 @@ export const SignInFactorTwoBackupCodeCard = (props: SignInFactorTwoBackupCodeCa
     isRequired: true,
   });
 
-  const handleBackupCodeSubmit: React.FormEventHandler = async e => {
+  const handleBackupCodeSubmit: React.FormEventHandler = e => {
     e.preventDefault();
     return signIn
       .attemptSecondFactor({ strategy: 'backup_code', code: codeControl.value })
       .then(res => {
         switch (res.status) {
           case 'complete':
-            return setActive({ session: res.createdSessionId, beforeEmit: navigateAfterSignIn });
+            return navigate(`../reset-password-success?createdSessionId=${res.createdSessionId}`);
```

Comment:
Should we use the URL api here?

Reply:
I don't think the URL api generates relative paths. Should we use `URLSearchParams` to construct the query string instead ?